src/components/BootstrapDialogTitle.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import DialogTitle from '@mui/material/DialogTitle';
+
+const BootstrapDialogTitle = ({ children, onClose, ...other }) => {
+    BootstrapDialogTitle.propTypes = {
+      children: PropTypes.node,
+      onClose: PropTypes.func.isRequired,
+    };  
+    return (
+    <DialogTitle sx={{ m: 0, p: 2 }} {...other}>
+      {children}  
+      {onClose ? (
```

Comment:
why we are checking if we have onClose when on close is defined as the required prop? So we will always have onClose